torch/_torch_docs.py

```diff
@@ -9566,9 +9568,10 @@ def merge_dicts(*dicts):
 Functions that return indices along a dimension, like :func:`torch.argmax` and :func:`torch.argsort`,
 are designed to work with this function. See the examples below.
 
-.. note::
-    This function is similar to NumPy's `take_along_axis`.
-    See also :func:`torch.gather`.
+.. seealso::
+        NumPy's `take_along_axis` which accepts an axis argument along which 1-D slices are taken.
```

Comment:
Should I just delete this then? Remove it from the seealso?

Reply:
It would actually be cool to call out the NumPy operators that are operations are based on but we don't have a template for it (yet) so yep, deleting this for now seems good.